src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -106,27 +105,44 @@
   (ct/report reports))
 
 (defmacro checking
-  "A macro intended to replace the testing macro in clojure.test with a
+  ^{:doc      "A macro intended to replace the testing macro in clojure.test with a
```

Comment:
I don't think this metadata will be effective unless you delete the `^` character on this line